site/content/guides/openapi-swagger.md

```diff
@@ -0,0 +1,230 @@
+---
+title: OpenAPI/Swagger Monitoring
+description: >-
+  OpenAPI and Swagger help users design and document APIs in a way that is readable from both humans and machines. As a consequence, they can also be used to generate the code that will run the specified API - both on the provider and consumer side. Can we leverage this same principle to simplify API monitoring?
+author: Giovanni Rago
+avatar: 'images/avatars/giovanni-rago.png'
+---
+
+## The OpenAPI Specification
+
+The [OpenAPI Specification (OAS)](https://spec.openapis.org/oas/v3.1.0) specifies a standard, language-agnostic and machine-readable format to describe a web API in one or more files. Nowadays, it acts as a vendor-neutral standard for describing the structure and behaviour of HTTP-based APIs, and exists as part of the [OpenAPI Initiative (OAI)](https://www.openapis.org/).
```

Comment:
Do we need an intro? Something like.
First, we are looking at OpenApi and afterward at how to kickstart your monitoring with existing OpenApi specs.

Reply:
Thanks, added this.